source/Halibut.Tests/Util/PortForwarderUtil.cs

```diff
@@ -0,0 +1,12 @@
+using Halibut.Tests.Util.TcpUtils;
+
+namespace Halibut.Tests.Util
+{
+    public static class PortForwarderUtil
```

Comment:
What purpose does this serve?

Reply:
Consistent logs and less code in each tests.

Otherwise each test will need to new up their own logger and pass it in.

Reply:
Isn't that the idea of the `ClientServiceBuilder` to abstract away all the plumbing. There is a bit of legacy code in some tests that could be cleaned up but wondering if we are just adding more things, `PortForwarderBuilder`, `PortForwarderUtil` etc for people to have to understand the usage of

Reply:
Yeah this could be removed when:
- The ClientServiceBuilder actually makes the port forwarder, currently the caller has to build it.
- All tests use that.

That could be done in a future PR and this can be deleted.